frontend/src/pages/HomePage/HomePage.js

```diff
@@ -1,51 +1,81 @@
 import React from 'react'
 
 import Grid from '@mui/material/Grid';
-import Box from '@mui/material/Box';
 
 import { Event } from './Event'
 import { ArtItem } from './ArtItem'
 import { ForumOverview } from './ForumOverview'
 
 class HomePage extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            error: null,
+            isLoaded: false,
+            artitems: [],
+            events: []
+        }
+    }
+
+    componentDidMount() {
+        const fetchHeaders = {
+            Authorization: 'Bearer eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJ1c2VyMSIsImF1dGhvcml0aWVzIjpbImFydGlzdCJdLCJleHAiOjE2OTg2MjQwMDB9.yXtKaBo2Nxt28f2U2wFwlRPG7HDlc91BSeObt-6UzLM'
+        }
```

Comment:
We shouldn't have hardcoded tokens here. You should import the token with `const { token } = useAuth()`. In order to use `useAuth`, you should import it like this: `import { useAuth } from "../auth/useAuth";`

See userCard component for reference.

For testing until we get the login page, you can call import `import { saveToken } from "../auth/useAuth";` and call `saveToken("your token")` before accessing the token.